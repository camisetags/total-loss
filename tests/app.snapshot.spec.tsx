import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import App from '../src/app/app';

describe('App wrapper', () => {
  it('should render app wrapper component', () => {
    const rendered = renderer.create(
      <MemoryRouter>
        <App />
      </MemoryRouter>,
    );

    expect(rendered.toJSON()).toMatchSnapshot();
  });

  it('should redirect to deck-select', () => {
    const rendered = renderer.create(
      <MemoryRouter initialEntries={['/deck-select']}>
        <App />
      </MemoryRouter>,
    );

    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
