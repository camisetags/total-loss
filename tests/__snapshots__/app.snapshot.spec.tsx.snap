// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App wrapper should redirect to deck-select 1`] = `
<main
  className="container"
>
  <div
    className="loading"
  >
     Carregando... 
  </div>
</main>
`;

exports[`App wrapper should render app wrapper component 1`] = `
<main
  className="container"
>
  <div>
    <h1
      className="main-title"
    >
       Me drunk 
    </h1>
    <div
      className="center-grid"
    >
      <a
        href="deck-select"
        onClick={[Function]}
      >
        <button
          className="button big"
        >
          Jogar!
        </button>
      </a>
    </div>
  </div>
</main>
`;
